docs/schedule-key-swap.md

```diff
@@ -0,0 +1,122 @@
+- If you installed Bittensor 6.12.2 not via PyPi package but directly from GitHub source, then do not use this feature. **Only the PyPi package of Bittensor 6.12.2 is malicious.**
+
+### Use this feature only if 
+
+Use this feature ONLY: 
+
+- If you downloaded the Bittensor PyPi package for the Bittensor version 6.12.2 and then performed any of the below operations.
+
+  **Impacted btcli 6.12.2 operations**
+
+  ```bash
+  btcli stake add
+  btcli stake remove
+  btcli wallet transfer
+  btcli root delegate
+  btcli root undelegate
+  btcli root set_take
+  btcli subnet register
+  ```
+
+:::important reminder
+If you installed Bittensor 6.12.2 not via PyPi package but directly from GitHub source, then do not use this feature. Only the PyPi package of Bittensor 6.12.2 is malicious.
+:::
+
+## Installing the schedule coldkey swap feature
+
+The schedule coldkey swap feature is available only at the below specified subtensor and bittensor repo branches:
+
+- **Subtensor repo**: Make sure to checkout the tag `specify release tag` on the Subtensor repo `https://github.com/opentensor/subtensor`.
+- **Bittensor repo and branch**: 
+    1. First make sure you do `git pull master` to pull the latest changes from the Bittensor repo `https://github.com/opentensor/bittensor`. 
+    2. Next, switch to `release/7.3.0` branch. 
+    3. Then install Bittensor by running the below command:
+    ```bash
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/opentensor/bittensor/release/7.3.0/scripts/install.sh)"
+    ```
+    4. Confirm that you have the correct Bittensor version installed by running the below command: 
+    ```bash
+    btcli --help
+    ```
+    You should see the version `7.3.0` in the output.
+---
+
+## Detailed steps using `btcli`
+
+### Syntax
+
+```bash
+btcli wallet schedule_coldkey_swap [--new_coldkey <new_coldkey_ss58_address>]
+```
+This command is used to **schedule** a swap your coldkey to a new coldkey. You must specify the new coldkey address. You must first separately create a new coldkey before using it in this command. 
+
+- To create a new coldkey, see [Creating a coldkey using `btcli`](./getting-started/wallets.md#creating-a-coldkey-using-btcli).
+- To see the SS58 address of your coldkey, see [Listing all the local wallets](https://docs.bittensor.com/getting-started/wallets#listing-all-the-local-wallets).
+
+The command checks for the validity of the new coldkey and prompts for confirmation before proceeding with the scheduling process.
+
+### Important 
+
+- This command does not immediately swap the coldkeys. It will schedule the coldkey swap event after a delay of 72-hours from the time this command is run. 
+- The actual coldkey swap event occurs on-chain when the chain is back to normal operations.
+- This is a free transaction. **However, you need a balance of at least one TAO in your old coldkey to initiate a coldkey swap to your new coldkey.**
```

Comment:
Will this work on testnet?  (can I add the --subtensor.network test flag)

Also, for what it's worth, when the price of TAO was very high, or for people earning through (say) HIP, having a whole TAO is a big ask.

Also: is this one TAO **total**, or one TAO **free on the coldkey**? (If the latter, see above for sure)